test/TUTORIAL.md

```
@@ -0,0 +1,1276 @@
+cd go-controller && make && cd ..
+```
+
+---
+
+## Architecture Overview
+
+Understanding where each piece lives is essential before making changes. The test
+infrastructure has six layers, and depending on your scenario, you may touch some
+or all of them:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)